tests/copy_test.go

```diff
@@ -131,3 +132,16 @@ func TestExplicitConversion(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func TestCopyStructByReflect(t *testing.T) {
+	type Info struct {
```

Comment:
```suggestion
func TestCopyStructByReflect(t *testing.T) {
	// https://github.com/gopherjs/gopherjs/issues/1156
	type Info struct {
```

For such corner cases I usually add a reference to the original issue in the test case, this helps with code archeology a year or two later :)